content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,355 @@
+for more information on configuring SPIRE to integrate with Istio deployments.
+
+#### SPIRE CA Integration Prerequisites
+
+These configuration requirements are necessary to successfully integrate Istio with SPIRE:
+
+1. Access [SPIRE Agent reference](https://spiffe.io/docs/latest/deploying/spire_agent/#agent-configuration-file) and
+   configure the SPIRE Agent socket path to match the Envoy SDS defined socket path.
+
+{{< text plain >}}
+socket_path = "/run/secrets/workload-spiffe-uds/socket"
+{{< /text >}}
+
+2. Share the SPIRE Agent socket with the pods within the node by deploying the
+   [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi).
+
+3. See [Install Istio](#install-istio) to configure Istio to integrate with the SPIFFE CSI Driver.
+
+## Install Istio
+
+1. [Download the latest Istio release](https://istio.io/latest/docs/setup/getting-started/#download). 
+
+2. After successfully [deploying SPIRE](#install-spire) into your environment, install Istio with custom patches for istio-ingressgateways/egressgateways as well as the istio-proxy.
+
+{{< text bash >}}
+$ istioctl install --skip-confirmation -f - <<EOF
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+spec:
+  profile: default
+  meshConfig:
+    trustDomain: example.org
+  values:
+    global:
+    # This is used to customize the sidecar template
+    sidecarInjectorWebhook:
+      templates:
+        spire: |
+          spec:
+            containers:
+            - name: istio-proxy
+              volumeMounts:
+              - name: workload-socket
+                mountPath: /run/secrets/workload-spiffe-uds
+                readOnly: true
+             volumes:
+               - name: workload-socket
+                 csi:
+                   driver: "csi.spiffe.io"
+  components:
+    ingressGateways:
+      - name: istio-ingressgateway
+        enabled: true
+        label:
+          istio: ingressgateway
+        k8s:
+          overlays:
+            - apiVersion: apps/v1
+              kind: Deployment
+              name: istio-ingressgateway
+              patches:
+                - path: spec.template.spec.volumes.[name:workload-socket]
+                  value:
+                    name: workload-socket
+                    csi:
+                      driver: "csi.spiffe.io"
+                - path: spec.template.spec.containers.[name:istio-proxy].volumeMounts.[name:workload-socket]
+                  value:
+                    name: workload-socket
+                    mountPath: "/run/secrets/workload-spiffe-uds"
+                    readOnly: true
+EOF
+{{< /text >}}
+
+3. Use [Automatic sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection)
+to automatically inject the `spire` template in the configuration above into new workloads pods.

```

Comment:
Aren't we suppose to mention both ways of injection? I've seen this across other examples;

I think this is a good place to mention the requirement on the injection template feature:
```suggestion
to automatically inject the `spire` template in the configuration above into new workloads pods that use the [proper notation for it](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#custom-templates-experimental).
```